schematic/models/validate_attribute.py

```diff
@@ -731,6 +769,31 @@ class ValidateAttribute(object):
     def __init__(self, dmge: DataModelGraphExplorer) -> None:
         self.dmge = dmge
 
+    def _login(
+        self,
+        access_token: Optional[str] = None,
+        project_scope: Optional[list[str]] = None,
+        columns: Optional[list] = None,
+        where_clauses: Optional[list] = None,
+    ):
+        # if the ValidateAttribute object already has a SynapseStorage object, just requery the fileview, if not then login
+        if hasattr(self, "synStore"):
```

Comment:
Why this and not `if self.synStore:`?

Reply:
This really should be functionality in synapse store class.

Reply:
> This really should be functionality in synapse store class.

@andrewelamb can you explain what you're envisioning for implementing this functionality in the `SynapseStorage` class? I just want to make sure we're on the same page. Thanks.

Reply:
> > This really should be functionality in synapse store class.
> 
> @andrewelamb can you explain what you're envisioning for implementing this functionality in the `SynapseStorage` class? I just want to make sure we're on the same page. Thanks.

This whole method is kind weird. I don't think I understand the code well enough to offer an alternative, but alof of this seems like code that should live in the Synapse Store class not here.

Reply:
I think given the scope of this PR and new feature its best to leave it as is. I had explored adding similar functionality to the `SynapseStorage` class as part of the refactor, so if necessary it can be added then.

Reply:
> Why this and not `if self.synStore:`

@andrewelamb because if the `ValidateAttribute` class does not have a `synStore` attribute then it will raise an error